style/blockly.css

```
@@ -263,7 +263,7 @@ body[data-theme="low-vision"] {
 .blockly-ws-search {
   background: var(--color-bg);
   margin-top: 5px;
-  border: solid var(--color-border-highlight) 4px;
+  /*border: solid var(--color-border-highlight) 4px;*/
```

Stylelint (17.11.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)